common/utilities/stopwatch.h

```diff
@@ -0,0 +1,47 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+// Helper classes to keep track of time
+#pragma once
+
+#include <chrono>
+namespace time_utilities
+{
+    class stopwatch
+    {
+    public:
+        using clock = std::chrono::steady_clock;
```

Comment:
Not needed

Reply:
Removed from other classes and placed on stopwatch file under the namespace, it's the only occurrence.